desc/compute/utils.py

```diff
@@ -69,27 +80,31 @@ def compute(names, params, transforms, profiles, data=None, **kwargs):
         Computed quantity and intermediate variables.
 
     """
+    p = _parse_parameterization(parameterization)
     if isinstance(names, str):
         names = [names]
     for name in names:
-        if name not in data_index:
+        if name not in data_index[p]:
             raise ValueError("Unrecognized value '{}'.".format(name))
```

Comment:
should we add to this error `"For parameterization {p}"` ?
Since in the future, if I understand correctly, we are thinking of allowing things to have like `FourierRZSurface.compute("n_rho")`, so if someone instead did `FourierRZSurface.compute("|F|")`, the error should specify that "|F|" is an unrecognized value specifically for the given object ( `FourierRZSurface`)that is being asked to compute it